### PR TITLE
Brandon/login legacy

### PR DESCRIPTION
**Pull request by brosscode**
Theme for XML and includes a login Legacy module

### CODE REVIEW

File: composeCore/src/main/java/com/brandon/composecore/theme/Type.kt [hunk 1]
```diff
@@ -7,15 +7,15 @@ import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
-import com.example.composecore.R
+import com.brandon.uicore.R as uiR
```
Comment (ForzenAcademy) [inline composeCore/src/main/java/com/brandon/composecore/theme/Type.kt]: Only alias if necessary

File: core/src/main/java/com/brandon/di/NetworkModule.kt [hunk 2]
```diff
@@ -7,6 +7,7 @@ import com.brandon.data.ForzenDao
 import com.brandon.data.ForzenDatabase
 import com.brandon.network.ForzenApiService
 import com.brandon.network.ForzenApiService.Companion.FORZEN_BASE_URL
+import com.brandon.network.ForzenApiServiceAlwaysFailsMock
```
Comment (ForzenAcademy) [inline core/src/main/java/com/brandon/di/NetworkModule.kt]: unused

File: createAccountCore/src/main/java/com/brandon/createaccount/core/data/CreateUserRepositoryImpl.kt [hunk 3]
```diff
@@ -5,6 +5,7 @@ import com.brandon.network.ForzenApiService
 import com.brandon.network.createuser.CreateUserResponse
 import com.google.gson.Gson
 import retrofit2.Response
+import retrofit2.converter.gson.GsonConverterFactory
```
Comment (ForzenAcademy) [inline createAccountCore/src/main/java/com/brandon/createaccount/core/data/CreateUserRepositoryImpl.kt]: unused

File: loginLegacy/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (ForzenAcademy) [inline loginLegacy/src/main/AndroidManifest.xml:2]: Condense

File: loginLegacy/src/main/res/layout-land/layout.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ForzenAcademy) [inline loginLegacy/src/main/res/layout-land/layout.xml]: Why empty layout?

File: uiCore/src/main/res/values/styles.xml [hunk 6]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="field_error_text">
```
Comment (ForzenAcademy) [inline uiCore/src/main/res/values/styles.xml]: Style names should be camel case

Comment (ForzenAcademy) [reply uiCore/src/main/res/values/styles.xml]: So change them all please

File: uiCore/src/main/res/values/themes.xml [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.ForzenBook.Light" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <item name="primary">@color/white</item>
```
Comment (ForzenAcademy) [inline uiCore/src/main/res/values/themes.xml:5]: Attribute and dimen names are final as camel case also

File: composeCore/src/main/java/com/brandon/composecore/theme/Type.kt (hunk 1 above)
Comment (brosscode) [reply composeCore/src/main/java/com/brandon/composecore/theme/Type.kt]: fixed

File: core/src/main/java/com/brandon/di/NetworkModule.kt (hunk 2 above)
Comment (brosscode) [reply core/src/main/java/com/brandon/di/NetworkModule.kt]: removed

File: createAccountCore/src/main/java/com/brandon/createaccount/core/data/CreateUserRepositoryImpl.kt (hunk 3 above)
Comment (brosscode) [reply createAccountCore/src/main/java/com/brandon/createaccount/core/data/CreateUserRepositoryImpl.kt]: removed

File: loginLegacy/src/main/AndroidManifest.xml (hunk 4 above)
Comment (brosscode) [reply loginLegacy/src/main/AndroidManifest.xml:2]: has stuff in it now

File: loginLegacy/src/main/res/layout-land/layout.xml (hunk 5 above)
Comment (brosscode) [reply loginLegacy/src/main/res/layout-land/layout.xml]: no longer empty

File: uiCore/src/main/res/values/styles.xml (hunk 6 above)
Comment (brosscode) [reply uiCore/src/main/res/values/styles.xml]: now are all camel case

File: uiCore/src/main/res/values/themes.xml (hunk 7 above)
Comment (brosscode) [reply uiCore/src/main/res/values/themes.xml:5]: renamed them

**Review by ForzenAcademy — APPROVED** [review]

**Pull request merged**